pages/logTempForm.js

```diff
@@ -2,8 +2,8 @@ import LogTemperatureForm from "@/components/Forms/LogTempForm/LogTempForm";
 
 export default function logTempForm() {
   return (
-    <main>
+    <div>
       <LogTemperatureForm />
-    </main>
+    </div>
```

Comment:
Do you really need a `<div>` for this case? Hint: Your `<div>`is surrounding just one element.